packages/contracts-bedrock/test/setup/Setup.sol

```diff
@@ -231,6 +233,7 @@ contract Setup {
         labelPredeploy(Predeploys.ETH_LIQUIDITY);
         labelPredeploy(Predeploys.OPTIMISM_SUPERCHAIN_ERC20_FACTORY);
         labelPredeploy(Predeploys.OPTIMISM_SUPERCHAIN_ERC20_BEACON);
+        labelPredeploy(Predeploys.NONCE_MANAGER);
```

Comment:
Don't we have to label the Preinstall contracts that we have deployed? i.e. BasicPaymaster...